understanding/20/extended-audio-description-prerecorded.html

```diff
@@ -12,11 +12,12 @@ <h1>Understanding Extended Audio Description (Prerecorded)</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Videos can be played with more detailed audio descriptions</dd>
-         <dt>Author task</dt><dd>Provide extended spoken descriptions of the visual content in videos</dd>
+         <dt>Goal</dt><dd>Videos can be played with more detailed audio descriptions.</dd>
+         <dt>What to do</dt><dd>Provide extended spoken descriptions of the visual content in videos.</dd>
+         <dt>Why it's important</dt><dd>People who are blind or have low vision can hear more about visual content while playing videos.</dd> 
```

Comment:
@rainbreaw commented
> Not a concern, but a recommendation for consideration, in understanding/20/audio-description-or-media-alternative-prerecorded.html, understanding/20/audio-description-prerecorded.html and understanding/20/extended-audio-description-prerecorded.html:

> Consider extending why it's important to not just include people who are blind or with low vision, but also  "and individuals with a cognitive disability that makes it difficult to understand implied meaning such as facial expressions."

Reply:
An attempt to do the same slight of hand I did in  [audio description or media alternative](https://github.com/w3c/wcag/pull/3356/files#r1330597329)
```suggestion
         <dt>Why it's important</dt><dd>People who are blind or who cannot understand the visual content can have it described to them while playing videos.</dd> 
```

Reply:
@mbgower this works for me.